scripts/qa_crowbarsetup.sh

```diff
@@ -1356,6 +1364,10 @@ EOF
         fi
     fi
 
+    if [[ $hcp = 1 ]]; then
+        add_hcp_repo
```

Comment:
nay, there will probably be other HCP repo versions and we might need to add it in other places, so it is nicer as a separate function